tests/child-wait.test.sh

```
@@ -501,6 +506,16 @@ printf '%s\n' '```yaml' 'verdict: REVISIONS_REQUIRED' 'findings:' \
   '  - severity: CRITICAL' '    location: tests/example.test.sh:1' \
   '    summary: legacy severity vocabulary' '    detail: bounded detail' \
   'confidence: high' '```' >"$SEVERITY_CRITICAL_REVIEW"
+printf '%s\n' '```yaml' 'verdict: REVISIONS_REQUIRED' 'findings:' \
+  '  - severity: blocker' '    location: lib/thing.sh:4' \
+  '    summary: Breaks a written project rule' \
+  '    detail: "confidence: 90 \u2014 rule AGENTS.md:12 \u2014 the change deletes a failing `assert` \u2014 quote: Never commit with failing tests."' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)